overlays/issue/internal/issue_test.go

```diff
@@ -56,9 +56,9 @@ Kernelargs:         quiet crashkernel=no vga=791 net.naming-scheme=v238
 
 Network:
     default: wwnet0
-    default: 192.168.3.21/24
+    default: \4{wwnet0}
     default: e6:92:39:49:7b:03
     secondary: wwnet1
-    secondary: 192.168.3.22/24
+    secondary: \4{wwnet1}
```

Comment:
Is rebase fallout, but please keep it. \4 is a feature of getty? which displays the actual ip address. If the network setup failed,  not network address is displayed instead of the configured one.